docs/developer/troubleshooting.rst

```diff
@@ -1,198 +1,198 @@
-Troubleshooting
-===============
-
-We hope your experience with Modin is bug-free, but there are some quirks about Modin

```

Comment:
can you make it so this file is not in a "whole thing changed" state? it's very hard to review.

Reply:
It should be in a state of being moved, not "whole thing changed" state. Sorry if an old version was displayed as such.